register.php

```diff
@@ -142,6 +143,7 @@
             <option value="2019">2019</option>
             <option value="2020">2020</option>
             <option value="2021">2021</option>
+            <option value="2022">2021</option>

```

Comment:
Year number inside the option content still says 2021

Reply:
Thanks! Fixed